### PR TITLE
Develop/nkyadav

### PR DESCRIPTION
**Pull request by Nkyadav650**
login integration & toastify message added

### CODE REVIEW

File: corsafe-poc-application/src/reducers/loginReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const loginUser = createAsyncThunk('login/loginUser', async ({ email, password }) => {
+   
+    return new Promise((resolve, reject) => {
+        setTimeout(() => {
+            if (email === 'test@example.com' && password === 'password') {
```
Comment (venkyEidiko) [inline corsafe-poc-application/src/reducers/loginReducer.jsx:7]: why here hardcoded data is there

File: corsafe-poc-service/src/main/java/com/aus/corsafe/service/UserRegisterServiceImpl.java [hunk 2]
```diff
@@ -46,4 +48,18 @@ public List<SecurityQuestion> getAllSecurityQuestion() {
        return securityQuestionRepository.findAll();
     }
 
+//    @Override
+//    public String addSecurityQuestionAnswerByUserId(List<SecurityQuestionKey> securityQuestionKey, Integer userId) {
+//      UserRegister userRegister = userRegisterRepo.findById(userId).orElseThrow(()->new RuntimeException("User does not exist !"));
+//      List<SecurityQuestionKey> list=userRegister.getSecurityQuestionList();
+//
```
Comment (venkyEidiko) [inline corsafe-poc-service/src/main/java/com/aus/corsafe/service/UserRegisterServiceImpl.java:55]: remove commented code and push

Comment (Nkyadav650) [reply corsafe-poc-service/src/main/java/com/aus/corsafe/service/UserRegisterServiceImpl.java]: ok removed commented code

File: corsafe-poc-application/src/reducers/loginReducer.jsx (hunk 1 above)
Comment (Nkyadav650) [reply corsafe-poc-application/src/reducers/loginReducer.jsx]: Ok I did enhancement

**Pull request merged**